Cyberpunk 2020/translations/es.json

```
@@ -1,211 +1,211 @@
-{
-	"accounting": "Accounting",
+Tirada de Localización{
```

```suggestion
{
```
remove wahtever this text bit is, otherwise things won't work